model/commitqueue/github_merge_generic.go

```diff
@@ -87,6 +87,9 @@ func (s *GithubMergePR) Send() (err error) {
 	defer cancel()
 	for i, pr := range s.PRs {
 		title := pr.CommitTitle
+		if s.MergeMethod == githubMergeMethodMerge {
+			title = ""
```

Comment:
I figured out from the ticket that the empty title using the default message, but I wouldn't have understood otherwise--can you put a comment?

Reply:
done